cardano_node_tests/tests/test_cli.py

```diff
@@ -393,6 +394,16 @@ def _check_stake_snapshot(  # noqa: C901
                 stake_snapshot = cluster_obj.g_query.get_stake_snapshot(
                     stake_pool_ids=expected_pool_ids
                 )
+
+                # check stake distribution on dbsync,
+                # the stake distribution is extracted from the set snapshot of the ledger.
```

Comment:
Please format this as two separate sentences (with capital letter etc.). Also please quote "set" for better understanding.